Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
Yeah, it definitely seems like calculate has a few problems in it. Fixing that is beyond the scope of fixing the tests for OSX though. Could you file another issue for fixing the calculate function?

---
In reply to: [74461625](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74461625) [](ancestors = 74461625,74359393,74356593,74354002,74342094,74336418)